server/views/pages/exhibition.njk

```diff
@@ -39,7 +39,9 @@
     <div class="container">
       <div class="grid">
         <div class="{{ {s:12, m:10, shiftM:1, l:7, xl:7} | gridClasses }} {{ {s:'HNL3', m:'HNL2'} | fontClasses}} {{ {s:4} | spacingClasses({margin: ['bottom']}) }}">
-          {% block exhibitionDescription %}{% endblock %}
+          {% block exhibitionDescription %}
```

Comment:
I guess this doesn't need to be in a `{% block %}` any more – that was in order to show something else in Cardigan before we had real content. Not that it matters/makes a difference, though.